ims/company/password_and_2fa.md

```diff
@@ -0,0 +1,46 @@
+# Password and 2FA policy
```

Comment:
Can we just use the full name here, rather than an acronym?

Reply:
2FA is a standard cybersecurity related acronym. However, for clarity I have expanded the acronym in the first line of P1 below